[stable/k8s-spot-termination-handler] Call for maintainers for charts with a single OWNER

mariusv opened the pull request:
this PR is part of #13867 and #13869

Signed-off-by: Marius Voila <myself@mariusv.com>

<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->
#### Is this a new chart
> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please host your own chart repository and submit your repository to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new chart repositories to the Helm Hub.

#### What this PR does / why we need it:

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

mariusv committed:
Call for maintainers for charts with a single OWNER

this PR is part of #13867 and #13869

Signed-off-by: Marius Voila <myself@mariusv.com>

tahmmee committed:
[stable/couchbase] Update to version 1.2.1 (#16905)

Updates Operator and Admission controller

Signed-off-by: Tommie McAfee <tommie@couchbase.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

eg-bot committed:
Update Express Gateway Images to v1.16.8 (#16942)

Signed-off-by: eg-bot <kevin+eg-bot@lunchbadger.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

amalsom10 committed:
[stable/redis-ha] Add a dedicated port in haproxy for redis slaves (#16824)

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

Signed-off-by: Amal Soman <amalsoman10@gmail.com>

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

Signed-off-by: Amal Soman <amalsoman10@gmail.com>

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

* [stable/redis-ha] Add a dedicated port in haproxy for redis slaves

Signed-off-by: Amal Soman <amalsoman10@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/gocd] Bump up GoCD Version to 19.8.0 (#16954)

* Bump up GoCD Version to 19.8.0

Signed-off-by: GoCD Build User <go-cd@googlegroups.com>

* Updated Changelog.

Signed-off-by: GoCD Build User <go-cd@googlegroups.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/phabricator] Release 7.0.5 updating components versions (#16952)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/mariadb] Release 6.8.8 updating components versions (#16955)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

omerlh committed:
add support for deployment labels (#16958)

Signed-off-by: omerlh <omerl@soluto.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/postgresql] Release 6.3.5 updating components versions (#16961)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/phpmyadmin] Release 3.0.4 updating components versions (#16962)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/mediawiki] Release 8.2.2 updating components versions (#16963)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/joomla] Release 6.1.4 updating components versions (#16965)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/suitecrm] Release 7.2.1 updating components versions (#16966)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/redmine] Release 12.2.3 updating components versions (#16967)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/prestashop] Release 8.2.1 updating components versions (#16968)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/parse] Release 9.2.2 updating components versions (#16969)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/ghost] Release 7.2.2 updating components versions (#16970)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/orangehrm] Release 6.2.2 updating components versions (#16971)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/drupal] Release 5.2.2 updating components versions (#16972)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/dokuwiki] Release 5.2.5 updating components versions (#16973)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/wordpress] Release 7.3.3 updating components versions (#16974)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/moodle] Release 6.1.6 updating components versions (#16975)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/odoo] Fix Ingress secret for multiple hosts (#16976)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

pikesley committed:
[stable/concourse] Bump `imageTag` to match actual released version (#16983)

Signed-off-by: pikesley <sam.pikesley@demandlogic.co.uk>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/openebs] Update charts to 1.2.0 release (#16959)

- Updated README
- Updated chart version 1.2.0
- Updated values.yaml
- Updated ENV variales for openebs apiserver
- Add tolerations for ndm daemonset
- Add cstorpoolclusters rules in validatingWebhookConfigurations

Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/owncloud] Release 6.2.2 updating components versions (#16964)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/odoo] Release 11.1.3 (#16979)

* [stable/odoo] Release 11.1.3 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Update postgresql dependency version

Signed-off-by: tompizmor <tompizmor@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/suitecrm] Release 7.2.2 updating components versions (#16991)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

rabbitstack committed:
[stable/sematext-agent] simplify volumes and volume mounts (#16993)

* simplify daemonset volumes

Signed-off-by: rabbitstack <rabbitstack7@gmail.com>

* bump chart version

Signed-off-by: rabbitstack <rabbitstack7@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

tompizmor committed:
[stable/redmine] Update dependencies version (#16992)

Signed-off-by: tompizmor <tompizmor@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

afirth committed:
[stable/prometheus-operator] Add override for Kubernetes Version (#16994)

* add override to sync_* and README

Signed-off-by: afirth <afirth@users.noreply.github.com>

* sync upstream dashboards and rules

Signed-off-by: afirth <afirth@users.noreply.github.com>

* delete old rules and dashboards no longer in upstream

Signed-off-by: afirth <afirth@users.noreply.github.com>

* update manually created dashboards

Signed-off-by: afirth <afirth@users.noreply.github.com>

* bump chart version

Signed-off-by: afirth <afirth@users.noreply.github.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

RemingtonReackhof committed:
[stable/memcached] Add updateStrategy configuration option (#16358)

* [stable/memcached] Add updateStrategy configuration option

Signed-off-by: Remington Reackhof <rreackhof@rmn.com>

* [stable/memcached] Set specific keys for updateStrategy to follow convention

Signed-off-by: Remington Reackhof <rreackhof@rmn.com>

* [stable/memcached] Fix README to reflect updateStrategy.type value requirement

Signed-off-by: Remington Reackhof <rreackhof@rmn.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

madddi committed:
[stable/velero] fix restic DaemonSet template syntax (#16960)

Signed-off-by: Mathias Merscher <Mathias.Merscher@dg-i.net>
Signed-off-by: Marius Voila <myself@mariusv.com>

phyrog committed:
[stable/prometheus-operator] Add conditional for prometheus.ingress.hosts (#16977)

* Add conditional for prometheus.ingress.hosts

Signed-off-by: Tom Gehrke <t.gehrke@reply.de>

* Bump patch version to 6.9.1

Signed-off-by: Tom Gehrke <t.gehrke@reply.de>

* Add condition for alertmanager as well

Signed-off-by: Tom Gehrke <t.gehrke@reply.de>
Signed-off-by: Marius Voila <myself@mariusv.com>

travisgroth committed:
[stable/pomerium] Scale out and docs improvements (#16338)

- Switch to a headless Authorize service to support client side
   load balancing
- Allow independent replicaCount scaling of the different services
- Add upgrading/changelog documentation and ToC

Signed-off-by: Travis Groth <tgroth@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/ghost] Release 7.2.3 updating components versions (#16999)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

runningman84 committed:
[stable/weave-scope] Fix error messages by using new architecture (#16796)

* Added cluster agent

Signed-off-by: Philipp <philipp.hellmich@bertelsmann.de>

* Fixed cluster agent

Signed-off-by: Philipp <philipp.hellmich@bertelsmann.de>

* Bump version

Signed-off-by: Philipp <philipp.hellmich@bertelsmann.de>

* Bump version

Signed-off-by: Philipp <philipp.hellmich@bertelsmann.de>

* Fix Readme

Signed-off-by: Philipp <philipp.hellmich@bertelsmann.de>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/phpmyadmin] Add miguelaeh to OWNERS (#17027)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/wordpress] Add miguelaeh to OWNERS (#17028)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/nats] Add miguelaeh to OWNERS (#17029)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/joomla] Add miguelaeh to OWNERS (#17030)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/testlink] Add miguelaeh to OWNERS (#17031)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/ghost] Add miguelaeh to OWNERS (#17032)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

BertrandGouny committed:
fixes #14976 reopened due to regression in PR #14981 (v1.0.4) (#16785)

Signed-off-by: Bertrand Gouny <bertrand.gouny@osixia.net>
Signed-off-by: Marius Voila <myself@mariusv.com>

jastBytes committed:
[stable/prometheus-operator] Add walCompression to CRD and bump chart version (#17004)

* Add walCompression to CRD and bump chart version to 6.8.4

Signed-off-by: jan.steffen <jan.steffen@figo.io>

* Add walCompression to CRD and bump chart version to 6.8.4

Signed-off-by: jan.steffen <jan.steffen@figo.io>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/prometheus-operator] walCompression of wrong type (#17046)

* Bump chart to version 6.9.3

Signed-off-by: jan.steffen <jan.steffen@figo.io>

* Remove quoting of walCompression in prometheus spec

Signed-off-by: jan.steffen <jan.steffen@figo.io>
Signed-off-by: Marius Voila <myself@mariusv.com>

cameronattard committed:
[stable/traefik] add maxIdleConnsPerHost (#16531)

Signed-off-by: Cameron Attard <cameron.attard@siteminder.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

jeremyxu2010 committed:
Fix haproxy check redis master configuration (#16709)

Signed-off-by: Jeremy Xu <jeremyxu2010@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

TommyLike committed:
Fix spark-operator charts issue (#16910)

Signed-off-by: TommyLike <hu.husheng@huawei.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

chrisjs committed:
[stable/spring-cloud-data-flow] Update configs for additional vars (#17050)

- add support for service labels/annotations, env var for cert trusting

Signed-off-by: Chris Schaefer <cschaefer@pivotal.io>
Signed-off-by: Marius Voila <myself@mariusv.com>

bonifaido committed:
traefik: reduce rbac scope if one namespace is handled (#16111)

Signed-off-by: Nandor Kracser <bonifaido@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

resouer committed:
Fix typo in title and better narrative (#15902)

A better narrative for #15004, and fix typo in title

Signed-off-by: Harry Zhang <lei.zhang@alibaba-inc.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

andrsp committed:
[stable/nginx-ingress] Add substitution for containerPorts from values (#17037)

* [stable/nginx-ingress] Add substitution for containerPorts from values

Use key-value range from .Values.controller.containerPort definition instead of mapping 1to1 http and https ports.

It allows passing extra configuration to nginx-ingress, such as validation-webhook with its own port, and define this port in values.
Without this option it’s not possible because of hardcoded http and https ports in deployment/daemonset templates.

Does not break anything.

Signed-off-by: Andrey Izotikov <andrsp@gmail.com>

* [stable/nginx-ingress] Bump version to 1.20.0

Signed-off-by: Andrey Izotikov <andrsp@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/rabbitmq] Add miguelaeh to OWNERS (#17026)

* [stable/rabbitmq] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/mongodb] Add miguelaeh to OWNERS (#17021)

* [stable/mongodb] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/external-dns] Add miguelaeh to OWNERS (#17023)

* [stable/external-dns] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/mariadb] Add miguelaeh to OWNERS (#17024)

* [stable/mariadb] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/odoo] Add miguelaeh to OWNERS (#17005)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/dokuwiki] Add miguelaeh to OWNERS (#17006)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/drupal] Add miguelaeh to OWNERS (#17007)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/osclass] Add miguelaeh to OWNERS (#17008)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/prestashop] Add miguelaeh to OWNERS (#17009)

* [stable/prestashop] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/redmine] Add miguelaeh to OWNERS (#17010)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/mediawiki] Add miguelaeh to OWNERS (#17011)

* [stable/mediawiki] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/kubewatch] Add miguelaeh to OWNERS (#17012)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/phpbb] Add miguelaeh to OWNERS (#17013)

* [stable/phpbb] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/orangehrm] Add miguelaeh to OWNERS (#17015)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/opencart] Add miguelaeh to OWNERS (#17016)

* [stable/opencart] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/postgresql] Add miguelaeh to OWNERS (#17017)

* [stable/postgresql] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/parse] Add miguelaeh to OWNERS (#17018)

* [stable/parse] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/suitecrm] Add miguelaeh to OWNERS (#17019)

* [stable/suitecrm] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/moodle] Add miguelaeh to OWNERS (#17020)

* [stable/moodle] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/jasperreports] Add miguelaeh to OWNERS (#17022)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/redis] Add miguelaeh to OWNERS (#17014)

* [stable/redis] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

meeshaeel committed:
Add command line parameters quoting in create connections script in Airflow chart (#17062)

Signed-off-by: Mikhail Cherepnev <mikhail.cherepnev@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

desaintmartin committed:
[stable/prometheus-operator] Update kube-state-metrics to 2.3.1 and grafana to 3.8.11. (#17057)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/phabricator] Add miguelaeh to OWNERS (#17025)

* [stable/phabricator] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

miguelaeh committed:
[stable/owncloud] Add miguelaeh to OWNERS (#17033)

* [stable/owncloud] Add miguelaeh to OWNERS

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

* Update Chart.yaml

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/falco] Performs a rollingUpdate when you change rules or settings (#17065)

Signed-off-by: Néstor Salceda <nestor.salceda@sysdig.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

azhelev committed:
[incubator/zookeeper] Fixes setting "zookeeper.log.threshold" option (#17066)

Signed-off-by: Atanas Zhelev <azhelev+github@mailbox.org>
Signed-off-by: Marius Voila <myself@mariusv.com>

fank committed:
Disable acme volume if kvprovider is configured) (#17067)

Signed-off-by: florian kinder <florian.kinder@mcl.de>
Signed-off-by: Marius Voila <myself@mariusv.com>

eddieramirez committed:
Fix typo on `values.yml` for `environmentSecrets` (#17052)

* should be `key` and not `value`
* per `.secretKeyRef.key` in `stable/atlantis/templates/statefulset.yaml`

Signed-off-by: Eddie Ramirez <eddieramirez@me.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

concaf committed:
Bump Ambassador to v0.78.0 (#17081)

Signed-off-by: Shubham <shubham@linux.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

caarlos0 committed:
[stable/prometheus-operator]: upgrade alertmanager (#17084)

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

caarlos0 committed:
[stable/traefik] backoff storeconfig job (#16513)

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

ryaneorth committed:
[stable/traefik] Updated acme storage location to be configurable (#16464)

* [stable/traefik] Updated acme storage location to be configurable

Signed-off-by: Ryan Orth <ryanorth@workfront.com>

* Fixed wrong version update
Signed-off-by: Ryan Orth <ryanorth@workfront.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

GreenKnight15 committed:
Fix space issue in volume mount (#17087)

Update spacing issue in deployment, fixed README, and updated values to have a better example.

Signed-off-by: James Ritter <jamesritter15@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

glowka committed:
[stable/influxdb] Add support for Azure to backup job (#16722)

Signed-off-by: Tomasz Glowka <glowka.tom@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

caarlos0 committed:
[stable/blackbox-exporter]: upgrade (#17083)

* [stable/blackbox-exporter]: upgrade

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>

* [stable/blackbox-exporter]: upgrade

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

jmistx committed:
[stable/grafana] Support statefulset as persistence option (#17063)

* [stable/grafana] make statefulset as persistence option

Signed-off-by: Oleg Blokhin <o.blohin@dodopizza.com>

* [stable/grafana] up chart version

Signed-off-by: Oleg Blokhin <o.blohin@dodopizza.com>

* [stable/grafana] fix name property of metadata

Signed-off-by: Oleg Blokhin <o.blohin@dodopizza.com>

* [stable/grafana] fix grafana version

Signed-off-by: Oleg Blokhin <o.blohin@dodopizza.com>

* Fix yaml lint error

Signed-off-by: Oleg Blokhin <o.blohin@dodopizza.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/mariadb] Release 6.8.10 updating components versions (#17086)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

sshahinco committed:
Added pod label capability to ClamAV (#16793)

Signed-off-by: Sherief Shahin <smshahin1611@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

adel-s committed:
Allow configuration using values for nodeSelector setting (#17091)

Signed-off-by: Adel Sachkov <adel.sachkov@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

happiness9721 committed:
fix: Add metrics labels and annotations only when enabled (#17060)

Signed-off-by: Joe Jiang <happiness9721@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/openEBS] Feature fix typo (#17098)

* correct typo of localProvisioner to localprovisioner

Signed-off-by: olivier beyler <olivier.beyler@orange.com>

* bump chart version

Signed-off-by: olivier beyler <olivier.beyler@orange.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

iNoahNothing committed:
[stable/ambassador] Major upgrades to Ambassador Pro (#16904)

* Upgrade Ambassador Pro to 0.7.0

- Ambassador Pro 0.7.0 introduces Ambassador certified builds that
  require
  - A license key
  - Use an image from the quay.io/datawire/ambassador_pro repository
- templates/deployment.yaml has been formatted in a backwards compatible
  way so all Ambassador Pro images before 0.7.0 will still use
Ambassador OSS repo.

Signed-off-by: Noah Krause <krausenoah@gmail.com>

* Store license key in k8s secret by default

Signed-off-by: Noah Krause <krausenoah@gmail.com>

* Add deprecation warning for using prom exporter

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>

* Major upgrades to Ambassador Pro installation

- Introduces the performance tuned and certified build of open source Ambassador, Ambassador core
- The license key is now stored and read from a Kubernetes secret by default
- Added `.Values.pro.licenseKey.secret.enabled` `.Values.pro.licenseKey.secret.create` fields to allow multiple releases in the same namespace to use the same license key secret.
- Introduces the ability to configure resource limits for both Ambassador Pro and it's redis instance
- Introduces the ability to configure additional `AuthService` options (see [AuthService documentation](https://www.getambassador.io/reference/services/auth-service/))
- The ambassador-pro-auth `AuthService` and ambassador-pro-ratelimit `RateLimitService` and now created as CRDs when `.Values.crds.enabled: true`
- Fixed misnamed selector for redis instance that failed in an edge case

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>

* Expose annotations for redis deploy

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>

* Minor updated to changelog and readme

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>

* Whitespace cleanup

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>

* Fix type error

Signed-off-by: Noah Krause <noahkrause@Noahs-MacBook-Pro.local>
Signed-off-by: Marius Voila <myself@mariusv.com>

msiuts committed:
[stable/k8s-spot-termination-handler] Update api version of daemonset (#16451)

* use apps/v1 as apiVersion for daemonset

Signed-off-by: Moritz Siuts <moritzs@moia.io>

* Bump patch version of k8s-spot-termination-handler

Signed-off-by: Moritz Siuts <moritzs@moia.io>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/parse] Release 9.2.4 updating components versions (#17104)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

alemorcuq committed:
Fix svc for redis with sentinel (#17095)

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

torstenwalter committed:
[stable/jenkins] display error when admin user does not exist. (#16949)

* Display error when admin user does not exist.

Print an error message when `master.sidecars.configAutoReload.enabled` is `true`, but the admin user which is needed to configure the SSH key does not exist.

Signed-off-by: Torsten Walter <mail@torstenwalter.de>

* only configure ssh key if user exists

Signed-off-by: Torsten Walter <mail@torstenwalter.de>
Signed-off-by: Marius Voila <myself@mariusv.com>

KealanM committed:
[incubator/kafka] Adding the ability to define a pod disruption budget for brokers (#15345)

* feature: adding the ability to define a pod disruption budget to the kafka brokers

Signed-off-by: Kealan Murphy <kealan.murphy@aspect.com>

* address PR comments

Signed-off-by: Kealan Murphy <kealan.murphy@aspect.com>

* bump chart version

Signed-off-by: Murphy, Kealan <kealan.murphy@aspect.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

jordanjennings committed:
Support frontend config values (#17074)

Signed-off-by: Jordan Jennings <jordanjennings@users.noreply.github.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

jordanjennings committed:
Update default dex app version to latest (#17117)

Signed-off-by: Jordan Jennings <jordanjennings@users.noreply.github.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
Change copyright to "The Helm Authors" (#16902)

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

to266 committed:
[stable/nfs-server-provisioner] fix deployment when not creating storage class (#16331)

* When a storage class is not created, mountOptions and parameters are also not included in the file

Signed-off-by: Tomas Ostasevicius <t.ostasevicius@gmail.com>

* Bump nfs-server-provisioner version

Signed-off-by: Tomas Ostasevicius <t.ostasevicius@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

mfuxi committed:
[incubator/etcd] implement TLS/SSL for client and peers in etcd (#16741)

* added TLS/SSL ability #16569

Signed-off-by:Moty Fux <moty.fux@gmail.com>
Signed-off-by: Moty Fux <moty.fux@gmail.com>

* removed hardcoded protocol and bumped version in Chart.yaml #16569

Signed-off-by:Moty Fux <moty.fux@gmail.com>
Signed-off-by: Moty Fux <moty.fux@gmail.com>

* added some comments in README.md #16569

Signed-off-by:Moty Fux <moty.fux@gmail.com>
Signed-off-by: Moty Fux <moty.fux@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

jonnydford committed:
[stable/prometheus-rabbitmq-exporter] ServiceMonitor Port Bugfix (#15801)

* Update and rename monitor.yaml to servicemonitor.yaml

The service monitor currently tries to scrape the publish port, which isn't available on the Service.
I've changed the endpoint port for the service monitor to scrape from the services' 'rabbitmq-exporter' port.

I've also changed the name to servicemonitor.yaml to match other Helm charts.

Signed-off-by: Jonny Ford <jonnyford1@gmail.com>

* Update Chart.yaml

bumped chart version, bugfix

Signed-off-by: Jonny Ford <jonnyford1@gmail.com>

* Update Chart.yaml

Signed-off-by: Maor Friedman <maor.friedman@redhat.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

Lunik committed:
[stable/openldap] Add documentation for 'persistence.existingClaim' (#16922)

Signed-off-by: Lunik <lunik@tiwabbit.fr>
Signed-off-by: Marius Voila <myself@mariusv.com>

max-rocket-internet committed:
[stable/metabase] Increase livenessProbe timeout (#16877)

Signed-off-by: Max Williams <max.williams@deliveryhero.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

micahhausler committed:
[stable/prometheus] Added /nodes/metrics permission to prometheus server (#16394)

This change allows prometheus to scrape nodes directly, instead of
proxying all requests through the API server.

Signed-off-by: Micah Hausler <mhausler@amazon.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

tommyknows committed:
[stable/grafana] Fix indentation of pod template (#17130)

* fix indentation of template

Signed-off-by: Ramon Rüttimann <ramon.ruettimann@gmail.com>

* bump chart version

Signed-off-by: Ramon Rüttimann <ramon.ruettimann@gmail.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

torstenwalter committed:
[stable/jenkins] add master.JCasC.defaultConfig (#16929)

* add master.JCasC.defaultConfig and configure location

- JCasC configuration is stored in template `jenkins.casc.defaults`
  so that it can be used in `config.yaml` and `jcasc-config.yaml`
  depending on if configAutoReload is enabled or not

- Jenkins Location (URL) is configured to provide a startin point
  for the config

Signed-off-by: Torsten Walter <mail@torstenwalter.de>

* improve JCasC default configuration

JCasC default configuration includes:
  - Jenkins url
  - disableRememberMe: false
  - mode: NORMAL
  - numExecutors: {{ .Values.master.numExecutors }}
  - projectNamingStrategy: "standard"

Signed-off-by: Torsten Walter <mail@torstenwalter.de>

* JCasC kubernetes config

Signed-off-by: Torsten Walter <mail@torstenwalter.de>

* JCasC: crumbIssue and security.apiToken

Signed-off-by: Torsten Walter <mail@torstenwalter.de>
Signed-off-by: Marius Voila <myself@mariusv.com>

egeland commented on stable/k8s-spot-termination-handler/Chart.yaml:
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: The K8s Spot Termination handler handles draining AWS Spot Instances in response to termination requests.
 name: k8s-spot-termination-handler
-version: 1.4.3
+version: 1.4.4
```
Looks like you need to bump it
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/16946/pull-charts-e2e/1173406394401951744#0:build-log.txt%3A181

mariusv replied:
Looks like the Chart version changed after my PR. Should be fixed now

Commit:
[stable/phabricator] Release 7.0.7 updating components versions (#17131)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

desaintmartin committed:
[stable/rabbitmq] Metrics: use ipv4 to connect to rabbit. (#17092)

* [stable/rabbitmq] Metrics: use ipv4 to connect to rabbit.

Otherwise, it could try to access IPv6 which may not be available, causing:

Get http://localhost:15672/api/nodes?sort=: dial tcp [::1]:15672: connect: cannot assign requested address

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Make it configurable.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>
Signed-off-by: Marius Voila <myself@mariusv.com>

Commit:
[stable/odoo] Release 11.1.5 updating components versions (#17141)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Marius Voila <myself@mariusv.com>

mariusv committed:
bumping chart version

Signed-off-by: Marius Voila <myself@mariusv.com>

mariusv committed:
Merge branch 'master' into become_maintainer

egeland commented on stable/k8s-spot-termination-handler/Chart.yaml:
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: The K8s Spot Termination handler handles draining AWS Spot Instances in response to termination requests.
 name: k8s-spot-termination-handler
-version: 1.4.4
+version: 1.4.5
```
Current chart is at 1.4.6 now... 😅

maorfr committed:
Merge branch 'master' into become_maintainer
